rosetta-source/src/main/rosetta/product-asset-type.rosetta

```diff
@@ -635,9 +635,11 @@ type InflationRateSpecification extends FloatingRateSpecification: <"A data to:
 	initialIndexLevel number (0..1) <"Initial known index level for the first calculation period.">
 	fallbackBondApplicable boolean (1..1) <"The applicability of a fallback bond as defined in the 2006 ISDA Inflation Derivatives Definitions, sections 1.3 and 1.8.">
 	calculationMethod InflationCalculationMethodEnum (0..1) <"Indicates how to use the inflation index to calculate the payment (e.g. Ratio, Return, Spread). Added for Inflation Asset Swap">
-	calculationStyle InflationCalculationStyleEnum (0..1) <"Indicates the style of how the inflation index calculates the payment (e.g. YearOnYear, ZeroCoupon).
-">
+	calculationStyle InflationCalculationStyleEnum (0..1) <"Indicates the style of how the inflation index calculates the payment (e.g. YearOnYear, ZeroCoupon).">  
+    finalPrincipalExchangeCalculation FinalPrincipalExchangeCalculation (0..1) <"To be specified only for inflation products that embed a redemption payment such as an inflation linked asset swap.">
 
```

Comment:
Can this finalPrincipalExchangeCalculation be changed to be more generic? i.e. just remove mention of inflation swap in the comments?